test/18-startup.ts

```diff
@@ -0,0 +1,16 @@
+import { Supervisor } from '../src/supervisor';
```

Comment:
There's 2 files with `18-`, not sure if that's a problem but it surprised me slightly